contrib/IECoreUSD/src/IECoreUSD/PointInstancerAlgo.cpp

```diff
@@ -86,6 +86,19 @@ IECore::ObjectPtr readPointInstancer( pxr::UsdGeomPointInstancer &pointInstancer
 	Canceller::check( canceller );
 	PrimitiveAlgo::readPrimitiveVariable( pointInstancer.GetAngularVelocitiesAttr(), time, newPoints.get(), "angularVelocity" );
 
+	DataPtr cortexInvisIds = DataAlgo::fromUSD( pointInstancer.GetInvisibleIdsAttr(), time, true );
+	if( cortexInvisIds )
+	{
+		newPoints->variables["invisibleIds"] = IECoreScene::PrimitiveVariable( PrimitiveVariable::Constant, cortexInvisIds );
+	}
+
+	pxr::SdfInt64ListOp inactiveIdsListOp;
+    pointInstancer.GetPrim().GetMetadata( pxr::UsdGeomTokens->inactiveIds, &inactiveIdsListOp );
+	newPoints->variables["inactiveIds"] = IECoreScene::PrimitiveVariable(
```

Comment:
I wonder, should we avoid making this if the list is empty? Or was never authored? I _think_ we're doing the "never authored" thing for `invisibleIds`?

Reply:
I've now added a test that we get only the expected primitive variables, which required fixes for both inactiveIds and invisibleIds.

I'm confused by your comment "I think we're doing the "never authored" thing for invisibleIds?", since invisibleIds is just being added - do you mean that you expected fromUSD to return null for attributes that aren't authored? That isn't happening, I had to explicitly add a check for HasAuthoredValue for invisibleIds. It's kind of weird that this isn't needed for other attributes like angularVelocities - I guess the USD schema defines a fallback for invisibleIds but not for things like angularVelocities.